src/sns-manager.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Principal } from "@dfinity/principal";
+import { mock } from "jest-mock-extended";
+import { DeployedSns, SnsWasmService } from "../candid/sns-wasm.idl";
+import { SnsManagerCanister } from "./sns-manager";
+
+describe("Sns", () => {
+  it("should return the list of sns", async () => {
+    const sns: DeployedSns[] = [
+      {
+        root_canister_id: [Principal.fromText("ryjl3-tyaaa-aaaaa-aaaba-cai")],
```

Comment:
Would be great to have a list of dummy/mock principal ids for future use.

Reply:
Done